ajo/personalization-client-side/README.md

```diff
@@ -63,13 +76,50 @@ function sendDisplayEvent(decision) {
               scopeDetails: scopeDetails,
             },
           ],
+          propositionEventType: {
+            display: 1
+          },
+        },
+      },
+    },
+  });
+}
+```
+
+6. For code based experience campaigns, click interaction events must manually be sent to indicate when an element has been clicked. This is done via the `sendEvent` command.
```

Comment:
"click interaction events" => "interaction events"

"when an element has been clicked" => "when a user has interacted with the content"